src/Apps/Order/Routes/Shipping2/Shipping2.jest.tsx

```diff
@@ -669,7 +678,7 @@ describe("Shipping", () => {
           await saveAndContinue()
 
           expect(
-            screen.queryByText("This field is required")
+            screen.queryByText(/[\w\s]is required/)
```

Comment:
required messages are no customized per field

Reply:
Now that these messages are customized, I think we should use specific error messages (instead of a regex) in each test.

For example, here we want to make sure that neither "Postal code is required" nor "State/Region/Provice is required" is displayed.

Reply:
Sadly you are right. Will update.